src/transformers/generation/logits_process.py

```diff
@@ -303,6 +304,8 @@ class RepetitionPenaltyLogitsProcessor(LogitsProcessor):
         penalty (`float`):
             The parameter for repetition penalty. 1.0 means no penalty. Above 1.0 penalizes previously generated
             tokens. Between 0.0 and 1.0 rewards previously generated tokens.
+        input_ids_seq_length (`int`, *optional*, defaults to 0):
```

Comment:
Let's rename it to something like `prompt_ignore_length`, so that it is clear that it modifies the original behavior.

`input_ids_seq_length`, at a first glance, looks like an input we should provide to get the correct behavior, as opposed to a modified behavior.

Let's also document the flag accordingly (it's an optional modifier), and explain that if it is to be used the class must be passed as a custom processor to generate. Ideally, with an example in the doctest below :D